daemons/gptp/common/avbts_port.hpp

```diff
@@ -202,11 +207,95 @@ class PortIdentity {
 typedef std::map < PortIdentity, LinkLayerAddress > IdentityMap_t;
 
 /**
+ * Structure for initializing the IEEE1588 class
+ */
+typedef struct {
+	/* clock IEEE1588Clock instance */
```

Comment:
Hmm, I didn't catch that because I usually have JAVADOC_AUTOBRIEF turned on in the Doxygen config (and it looks like the Harman-STC folks do as well based on the avtp-pipeline Doxygen config), in which case the first line of text becomes the 'brief' description automatically.  We don't have that turned on for gptp, though, so you're correct that it won't get picked up as a Doxygen comment.  I'll fix them.